### PR TITLE
External licensing

### PR DESCRIPTION
**Pull request by hiergaut**
Added license and readme to all externals

### CODE REVIEW

**Comment by hiergaut** [comment]
@nmellado ready to merge

File: CHANGELOG [hunk 1]
```diff
@@ -10,6 +10,9 @@ Current head v.2.1.0 rc
   * Remove first server impl (#16)
 - CMake
   * Remove dead code (#15)
+  * Removed first server impl (#16)
```
Comment (nmellado) [inline CHANGELOG:13]: Duplicated line (see line 10)

**Review by nmellado — CHANGES_REQUESTED** [review]
All good, except a minor issue with CHANGELOG

**Comment by nmellado** [comment]
One last thing, can you squash the two CHANGELOG commits please ?

File: CHANGELOG (hunk 1 above)
Comment (hiergaut) [reply CHANGELOG]: Oups, I didn't pay attention when managing the conflict. Resolved below.

**Comment by hiergaut** [comment]
Done.

**Review by nmellado — APPROVED** [review]

**Pull request merged**